src/components/layout/livechat.js

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import Cookies from 'js-cookie'
 import styled from 'styled-components'
+import * as CustomerSDK from '@livechat/customer-sdk'
```

Comment:
can we only import the function we need 

```suggestion
import { init } from '@livechat/customer-sdk'
```